FlorenceBML/BMLacq679/BMLacq679.xml

```diff
@@ -39,25 +39,91 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <msContents>
                         <summary/>
                         <msItem xml:id="ms_i1">
-                            <locus from="2r" to="102v"/>
+                            <locus from="2r" to="103v"/>
                             <title ref="LIT2000Mazmur" type="incomplete">Psalms 1-50</title>
                             <note>With pseudoepigraphic no. 151 in <locus from="102r" to="102v"/></note>
                         </msItem>
                         <msItem xml:id="ms_i2">
-                            <locus from="102v" to="114r"/>
+                            <locus from="103v" to="115r" type="complete"/>
                             <title ref="LIT1828Mahale">15 Songs of the Prophets</title>
                         </msItem>
```

Comment:
delete `</msItem>` here and close after item 15

Reply:
done.